src/EditorFeatures/CSharpTest/Diagnostics/GenerateMethod/GenerateMethodTests.cs

```diff
@@ -7246,7 +7246,7 @@ public void M1()
 parseOptions: TestOptions.Regular);
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateMethod)]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/15508"), Trait(Traits.Feature, Traits.Features.CodeActionsGenerateMethod)]
```

Comment:
>Skip = "https://github.com/dotnet/roslyn/issues/15508 [](start = 14, length = 53)

It feels wrong that parser now produces single type tuple types or tuples. I believe language grammar doesn't allow such syntax. I think it should add a missing comma token and missing type/expression for the second element, this is what we do for other similar situations.